### DISCUSSION TITLE
Role of the "Repeat" action in notifications

### DISCUSSION THREAD

**Discussion by Neluji** [FossifyOrg/General-Discussion#459] [Q&A] [answered]
Hi there, I was wondering what is happening when we press the "Repeat" button in a notification ? It is present for tasks & events, whether they are recurring or not 🤷🏻‍♂️

**Comment by Aga-C** [comment]
Fossify Calendar has only "Snooze" button in event notifications, there's no "repeat".

**Reply by Neluji** [reply]
Yes sorry, I have it in french so I translated literally but this is what I mean, the "Snooze" button ^^ Actually seeing it called like this I understand it better, it probably postpone the notification a few (how many ?) minutes later, so this is maybe just a translation issue

**Reply by Aga-C** [reply, marked as answer]
It will notify once again after some time. It's configurable in Settings, and the default is 10 minutes. You can also turn off snooze time in Settings, then it will ask after tapping the button what you want to do.

**Reply by Aga-C** [reply]
Can you fix the translation? Either on [Weblate](https://hosted.weblate.org/translate/fossify/commons/fr/?checksum=0abe3d706d76f882), or by editing a file on [GitHub](https://github.com/FossifyOrg/Commons/blob/master/commons/src/main/res/values-fr/strings.xml#L629) (line with `name="snooze"`).
